packages/react/src/components/Modal/Modal.js

```diff
@@ -297,7 +306,8 @@ export default class Modal extends Component {
         ref={this.innerModal}
         role="dialog"
         className={`${prefix}--modal-container`}
-        aria-label={modalAriaLabel}
+        aria-label={this.props['aria-label'] || modalAriaLabel}
+        aria-labelledby={this.props.id || null}
```

Comment:
Would we just want to support `aria-label`? I think `this.props.id` might trip it up since it's referring to the modal node in this context, which has `role="presentation"`.

Reply:
yeah that makes sense, removed `aria-labelledby`